update_actions.py

```diff
@@ -0,0 +1,59 @@
+"""
+  Scans directories for files that satisfy a specific mask and updates
+  the actions' tags.
+  python3 update_actions.py --source “one/*.y*ml two/*.y*ml three/*.y*ml”
+"""
+
+import glob
+import argparse
+import re
+from pathlib import Path
+from github import Github
+
+parser = argparse.ArgumentParser()
+cache = {}
+
+parser.add_argument(
+    '-s', '--sources',
+    type=str, help='Source directories to scan'
+)
+
+args = parser.parse_args()
+sources = args.sources.split(' ')
+g = Github()
+pattern = r"\s+action:.*"
```

Comment:
Maybe using a regexp to catch all params would be easier? Something along the lines of 

```python
pattern = r"\s+action:\s*(?P<svc>gh|github):(?P<org>[\w-]+)/(?P<repo>[\w-]+)@(?P<version>v[\d.]+)"
r = re.compile(pattern)

...
for file_path in glob.iglob(source):
  for line in open(file_path).readlines():
    match = r.match(line)
    if match:
       # use match['svc'], match['org'], match['repo'] and match['version']
```